docs/dag-run.rst

```diff
@@ -0,0 +1,193 @@
+in configuration file. When turned off, the scheduler creates a DAG run only for the latest interval.
+
+.. code:: python
+
+    """
+    Code that goes along with the Airflow tutorial located at:
+    https://github.com/apache/airflow/blob/master/airflow/example_dags/tutorial.py
+    """
+    from airflow import DAG
+    from airflow.operators.bash_operator import BashOperator
+    from datetime import datetime, timedelta
+
+
+    default_args = {
+        'owner': 'Airflow',
+        'depends_on_past': False,
+        'start_date': datetime(2015, 12, 1),
+        'email': ['airflow@example.com'],
+        'email_on_failure': False,
+        'email_on_retry': False,
+        'retries': 1,
+        'retry_delay': timedelta(minutes=5)
+    }
+
+    dag = DAG(
+        'tutorial',
+        default_args=default_args,
+        description='A simple tutorial DAG',
+        schedule_interval='@daily',
+        catchup=False)
+
+In the example above, if the DAG is picked up by the scheduler daemon on 2016-01-02 at 6 AM, 
+(or from the command line), a single DAG Run will be created, with an `execution_date` of 2016-01-01, 
+and the next one will be created just after midnight on the morning of 2016-01-03 with an execution date of 2016-01-02.
+
+If the ``dag.catchup`` value had been True instead, the scheduler would have created a DAG Run 
+for each completed interval between 2015-12-01 and 2016-01-02 (but not yet one for 2016-01-02, 
+as that interval hasn’t completed) and the scheduler will execute them sequentially. 
+
+Catchup is also triggered when you turn off a DAG for a specified period of time and then re-enable.
+
+This behavior 
+is great for atomic datasets that can easily be split into periods. Turning catchup off is great 
+if your DAG Runs perform backfill internally.
+
+
+Backfill
+---------
+There can be the case when you may want to run the dag for a specified historical period e.g. a data pipeline
+which dumps data in a DFS every day and another pipeline which requires last 1 month of data in DFS. 
+This is known as Backfill.
+
+You may want to backfill the data even in the cases when catchup is disabled. This can be done through CLI. 
+Run the below command
+
+.. code:: bash
+
+    airflow backfill -s START_DATE -e END_DATE dag_id
+
+The above command will re-run all the instances of the dag_id for all the intervals within the start date and end date.
+
+Re-run Tasks
+------------
+It can happen, that some of the tasks can fail during the scheduled run. Once you have fixed 
+the errors after going through the logs, you can re-run the tasks by clearing it for the 
+scheduled date. Clearing a task instance doesn't delete the task instance record. 
+Instead it updates ``max_tries`` to ``0`` and set the current task instance state to be ``None``, this forces the task to re-run.
+
+Select the failed task and click on **Clear**. This will clear the status of the task from 
```

Comment:
It's not clear from this where the Clear button is (I know what you mean, it's just not clear in this doc).

```suggestion
Select the failed task and click on **Clear** in the Task Instance modal. This will clear the status of the task from 
```

maybe? Or 
```suggestion
Click on the failed task in the Tree or Graph views and then click on **Clear**.
``